docs/docs/developer/pr-checklist.md

```diff
@@ -1,5 +1,6 @@
 # PR Checklist
 
+A minimal devcontainer is supplied with this repository. All commands can be executed directly inside this container to avoid tedious installation of the environment.
```

Comment:
Since you mention that you can't run the Compose stack in this dev container, it'd be good to clarify that in the docs here as well.

Reply:
Fixed: